examples/benchmarks/XGBoost/workflow_config_xgboost_Alpha360.yaml

```diff
@@ -19,19 +19,34 @@ data_handler_config: &data_handler_config
 port_analysis_config: &port_analysis_config
     strategy:
         class: TopkDropoutStrategy
-        module_path: qlib.contrib.strategy.strategy
+        module_path: qlib.contrib.strategy
         kwargs:
+            model: <MODEL>
+            dataset: <DATASET>
             topk: 50
             n_drop: 5
     backtest:
-        verbose: False
-        limit_threshold: 0.095
+        start_time: 2017-01-01
+        end_time: 2020-08-01
         account: 100000000
         benchmark: *benchmark
-        deal_price: close
-        open_cost: 0.0005
-        close_cost: 0.0015
-        min_cost: 5
+    executor: 
+        class: NestedExecutor
+        module_path: qlib.backtest.executor
+        kwargs: 
+            time_per_step: week
```

Comment:
Please discuss with  Xiaodong about the executor.
I don't think NestedExecutor is necessary

Reply:
ok